app/processing/delta/index.js

```diff
@@ -1,42 +1,35 @@
 const allocateToLedgers = require('./allocate-to-ledgers')
 const calculateLineDeltas = require('./calculate-line-deltas')
 const calculateOverallDelta = require('./calculate-overall-delta')
-const getDefaultLedger = require('./get-default-ledger')
 const getInvoiceLines = require('./get-invoice-lines')
 const getOutstandingLedgerValues = require('./get-outstanding-ledger-values')
 const zeroValueSplit = require('./zero-value-split')
+const copyPaymentRequest = require('./copy-payment-request')
 
 const calculateDelta = (paymentRequest, previousPaymentRequests) => {
   const invoiceLines = getInvoiceLines(paymentRequest, previousPaymentRequests)
 
   const lineDeltas = calculateLineDeltas(invoiceLines)
   const overallDelta = calculateOverallDelta(invoiceLines)
   const updatedPaymentRequest = copyPaymentRequest(paymentRequest, overallDelta, lineDeltas)
+  const deltaPaymentRequest = JSON.parse(JSON.stringify(updatedPaymentRequest))
```

Comment:
I think we need a better way of doing this, whether that's creating a reusable function to do so -- used 3 times throughout and likely to increase -- again, not in the scope of this ticket, more technical debt